src/components/RenderMissed.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+
+import ArticleBlock from './ArticleBlock';
+
+class RenderMissed extends Component {
+    render() {
+        let missedArticles=[];
+        for (let i = 0; i < 3; i++) {
+          let missArt = this.props.missedArticles[i];
+          missedArticles.push(
+            <ArticleBlock
```

Comment:
assign a 'key' to each component that will be added as a child to avoid react warnings